app/assets/stylesheets/buttons.scss

```diff
@@ -0,0 +1,21 @@
+// gradient buttons
+$gradient: linear-gradient(45deg, black, blue);
+$gradient2: linear-gradient(45deg, black, blue);
+
+.gbutton {
+  color: transparent;
```

Comment:
Properties should be ordered background, -webkit-background-clip, border, border-image, border-image-slice, color, font-size, letter-spacing, text-fill-color, transition